charts/rs-server-station-secrets/values.yaml

```diff
@@ -0,0 +1,169 @@
+# Copyright 2024 CS Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -- Namespace for the deployment
+namespace: processing
+
+app:
+  stations:
+    adgs:
+      service: 
+        name: auxip
+        url: http://mockup-station-adgs-svc.processing.svc.cluster.local:8080                    
+      domain: mockup-station-adgs-svc.processing.svc.cluster.local
+      authentication:
+        auth_type: oauth2
+        token_url: http://mockup-station-adgs-svc.processing.svc.cluster.local:8080/oauth2/token
+        grant_type: password
+        scope: null
+        username: test
```

Comment:
is it normal to have this values in clear, here?

Reply:
yes, they are programmatically overwritten